localnet/poktrolld/config/appgate_server_config.yaml

```diff
@@ -4,5 +4,11 @@ self_signing: true
 signing_key: app1
 # The host and port that the appgate server will listen on
 listening_endpoint: http://localhost:42069
-# tcp://<host>:<port> to a full pocket node for reading data and listening for on-chain events
-query_node_url: tcp://127.0.0.1:36657
+# Pocket node URL that exposes CometBFT JSON-RPC API.
+# This can be used by the Cosmos client SDK, event subscriptions, etc...
+query_node_rpc_url: tcp://poktroll-sequencer:36657
+# Pocket node URL that exposes the Cosmos gRPC service.
+tx_node_grpc_url: tcp://poktroll-sequencer:36658
```

Comment:
Does the `appgate` stuff need a `tx_node`? Doesn't it just require the `query_node`?